sthgetter/sthgetter.go

```diff
@@ -7,27 +7,30 @@ import (
 	"log"
 	"time"
 
+	ct "github.com/google/certificate-transparency-go"
 	"github.com/google/certificate-transparency-monitor/client"
+	"github.com/google/certificate-transparency-monitor/monitor"
+	"github.com/google/certificate-transparency-monitor/storage"
 )
 
 var logStr = "STH Getter"
 
 // Run runs an STH Getter, which periodically gets an STH from a Log, checks
 // that each one meets per-STH requirements defined in RFC 6962, and stores
 // them.
-func Run(ctx context.Context, lc *client.LogClient, url string, period time.Duration) {
+func Run(ctx context.Context, lc *client.LogClient, st storage.APICallWriter, url string, period time.Duration) {
 	log.Printf("%s: %s: Started with period %v", url, logStr, period)
 
 	t := time.NewTicker(period)
 	for {
 		select {
```

Comment:
Maybe not in this PR, but I feel that this "infinite loop" that takes a context should abort if the context is done? Add the appropriate clause in the `select`? Optional for this PR, though, could be just a TODO, or a mental note... :smiley:

Reply:
Done (I think... check I did it right?)

Reply:
I think so? :grin: